beanquery/shell.py

```diff
@@ -21,19 +21,20 @@
 
 import click
 
-from beanquery import query_parser
-from beanquery import query_compile
-from beanquery import query_env
-from beanquery import query_execute
-from beanquery import query_render
-from beanquery import numberify
 from beancount.parser import printer
 from beancount.core import data
 from beancount.utils import misc_utils
 from beancount.utils import pager
 from beancount.parser.version import VERSION
 from beancount import loader
 
+from beanquery import query_parser
+from beanquery import query_compile
+from beanquery import query_env
+from beanquery import query_execute
+from beanquery import query_render
+from beanquery import numberify
```

Comment:
FYI: third party imports should be placed before the local ones (`wrong-import-order` check)